### PR TITLE
added ITHC

### PR DESCRIPTION
**Pull request by darren221980**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPO-4392

### Change description ###

enabled ITHC

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by thomast1906 — APPROVED** [review]

File: environments/network/ithc.tfvars [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+]
+
+hub = "prod"
+
+additional_routes = [
+  {
+    name                   = "preview"
+    address_prefix         = "10.12.64.0/18"
+    next_hop_type          = "VirtualAppliance"
+    next_hop_in_ip_address = "10.11.72.36"
```
Comment (thomast1906) [inline environments/network/ithc.tfvars:26]: Is this correct for ITHC?

File: environments/network/ithc.tfvars [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  },
+  {
+    name                   = "10_0_0_0"
+    address_prefix         = "10.0.0.0/8"
+    next_hop_type          = "VirtualAppliance"
+    next_hop_in_ip_address = "10.11.8.36"
```
Comment (thomast1906) [inline environments/network/ithc.tfvars:32]: This is Production Palo - change to Nonprod

File: environments/network/ithc.tfvars [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    name                   = "192_168_0_0"
+    address_prefix         = "192.168.0.0/16"
+    next_hop_type          = "VirtualAppliance"
+    next_hop_in_ip_address = "10.11.8.36"
+  }
+]
+
+additional_routes_appgw = [
+  {
+    name                   = "core_infra_vnet_idam_ithc2"
```
Comment (thomast1906) [inline environments/network/ithc.tfvars:50]: Change all 3 to ithc equivalent

**Review by thomast1906 — CHANGES_REQUESTED** [review]

File: environments/network/ithc.tfvars (hunk 1 above)
Comment (darren221980) [reply environments/network/ithc.tfvars]: updated

File: environments/network/ithc.tfvars (hunk 2 above)
Comment (darren221980) [reply environments/network/ithc.tfvars]: updated

File: environments/network/ithc.tfvars (hunk 3 above)
Comment (darren221980) [reply environments/network/ithc.tfvars]: updated

**Review by thomast1906 — APPROVED** [review]

**Pull request merged**